code/game/objects/items/devices/chameleonproj.dm

```diff
@@ -115,9 +115,10 @@
 		M << "<span class='danger'>Your chameleon-projector deactivates.</span>"
 	master.disrupt()
 
-/obj/effect/dummy/chameleon/ex_act() //ok now THATS some serious protection against explosions right here
+/obj/effect/dummy/chameleon/ex_act(var/severity) //no longer bomb-proof
 	for(var/mob/M in src)
 		M << "<span class='danger'>Your chameleon-projector deactivates.</span>"
+		M.ex_act(severity)

```

Comment:
The `ex_act` should probably be in a `spawn` so it happens after the user is ejected from the dummy object.